src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -192,6 +192,13 @@ type Data = {
 
 const scoresAP = [1, 2, 3, 4, 5];
 const scoresIB = [1, 2, 3, 4, 5, 6, 7];
+const existingAP: Record<string, boolean> = {};
+// filter duplicate exam names and ones already selected
+reqsData.AP = reqsData.AP.filter(ap => {
+  const inExisting = ap.name in existingAP;
+  existingAP[ap.name] = true;
+  return !inExisting;
+});
```

Comment:
Why don't you do the same for IB?

Reply:
There are no duplicate exams for IB but I can do the check anyways